src/components/RoomView/gameLog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './roomView.css';
+import GuessesRemaining from './GuessesRemaining';
+
+interface ClueType {
```

Comment:
just noting here again that we'll be able to clean this up soon, as this interface is defined in multiple places!